serialization.rst

```diff
@@ -167,12 +167,16 @@ possible mapping values are:
 - *not set* or ``NULL`` — this is the default.
 
   - A BSON array will be deserialized as a PHP ``array``.
-  - A BSON document (root or embedded) without ``__pclass`` property [1]_ becomes a
-    PHP ``stdClass`` object, with each BSON document key becoming a
+  - A BSON document (root or embedded) without a ``__pclass`` property [1]_
+    becomes a PHP ``stdClass`` object, with each BSON document key set as a
     public ``stdClass`` property.
-  - A BSON document (root or embedded) with ``__pclass`` property [1]_ becomes
+  - A BSON document (root or embedded) with a ``__pclass`` property [1]_ becomes
     a PHP object of the class name as defined by the ``__pclass`` property.
 
+    If the named class does not exist, ``stdClass`` will be used and each BSON
```

Comment:
After reading https://jira.mongodb.org/browse/PHPC-330?focusedCommentId=966240&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-966240 — I withdraw this comment.